README.md

```diff
@@ -20,7 +20,17 @@ console.log(second_array) // [1, 2, 3]
 
 ```js
 function swap(arr1, arr2) {
-  // -- YOUR ANSWER HERE --
+ 
+  let bucket1 =arr1;
+  console.log("bucket1 " + bucket1)
+  arr1 =  arr2;
+  console.log("arr1" + arr1)
+  arr2=bucket1;
+  console.log("arr2" + arr2)
+  first_arr = arr1;
+  second_array = arr2;
+  return(arr1, arr2);
+
```

Comment:
This function does not need to return anything. It needs to use the fact that arrays are passed by reference to complete it.